api/importer/package_processor.go

```diff
@@ -21,6 +21,7 @@ type Renderer interface {
 type ImportPackage interface {
 	io.Closer
 	GetResource(resourceName string) (io.ReadCloser, error)
+	CheckIfResourceExists(resourceName string) error
```

Comment:
we can try to shorten the signature and not use the absence of error as a value

```
ResourceExists(resourceName string) (bool, error)
```

Reply:
Renamed it and added the second return value.